src/main/java/com/alienfast/bamboozled/ruby/config/RubyBuildConfigurationPlugin.java

```diff
@@ -85,6 +86,11 @@ public boolean isApplicableTo( Plan plan ) {
     }
 
     @Override
+    public boolean isApplicableTo(@NotNull ImmutablePlan plan) {
+        return false;
```

Comment:
Seems strange that this would be `false` which would make it show up on the `Misc` tab.